.github/workflows/push_tests.yml

```diff
@@ -137,7 +137,7 @@ jobs:
 
     - name: Run PyTorch CUDA tests
       env:
-        HF_TOKEN: ${{ secrets.HF_TOKEN }}
+        HF_TOKEN: ${{ secrets.DIFFUSERS_HF_HUB_READ_TOKEN }}
```

Comment:
In the `push_tests` we don't run the big checkpoint related stuff no?

Reply:
No we don't. Just Fast GPU tests.

Reply:
Hmm not a blocker but I think it can be removed then no as we don't need the token in these tests.

Reply:
For the single file model tests we check loading and config with a real checkpoint since it's challenging to create dummy checkpoints for single file models. 

Once we work out single file serialization we can remove.

Reply:
Ah okay. Let's go!